Python.gitignore

```diff
@@ -106,8 +106,9 @@ ipython_config.py
 #pdm.lock
 #   pdm stores project-wide configurations in .pdm.toml, but it is recommended to not include it
 #   in version control.
-#   https://pdm.fming.dev/#use-with-ide
+#   https://pdm.fming.dev/latest/usage/project/#working-with-version-control
 .pdm.toml
+.pdm-python
```

Comment:
Suggest to include `.pdm-build/` as referenced here: https://github.com/pdm-project/pdm/blob/main/src/pdm/cli/templates/default/.gitignore#L112

Reply:
Added although it is not mentioned in the docs.

Reply:
It should be OK as the template is used when generating a new project when using `pdm init`.